README.md

```diff
@@ -30,29 +30,23 @@ inference.
     ```
     sudo apt-get install libturbojpeg
     sudo apt-get install python3-tk
-    virtualenv ./toolbox_env -p python3
-    source ./toolbox_env/bin/activate
-    pip install -r requirements.txt
     ```
 
-3. Build COCO API
+3. Create virtual environment
 
     ```
-    cd external/cocoapi
-    2to3 . -w
-    cd PythonAPI
-    python setup.py build_ext --inplace
+    sh tools/init_venv.sh
```

Comment:
Looks like it is the BASH script. Why do you call sh instead of bash?

Reply:
done